SingularityUI/app/templates/tasksTable/tasksActiveBody.hbs

```diff
@@ -4,15 +4,15 @@
             <thead>
                 <tr>
                     <th data-sort-attribute="taskId.id">
-                        Name
+                        Task ID
                     </th>
                     <th class="hidden-xs" data-sort-attribute="taskId.startedAt">
-                        Started
+                        Started At
                     </th>
                     <th class="hidden-xs" data-sort-attribute="host">
                         Host
                     </th>
-                    <th class="hidden-xs" data-sort-attribute="taskId.rackId">
+                    <th class="visible-lg" data-sort-attribute="taskId.rackId">
```

Comment:
What was the reasoning for making this visible vs not and lg vs xs? I would argue that just seeing the task id, cpu and mem on a small device is the more important. And probably host over rack if we were to choose one more to add